relay-dynamic-config/src/global.rs

```diff
@@ -214,14 +214,6 @@ pub struct Options {
     )]
     pub span_extraction_sample_rate: Option<f32>,
 
-    /// Overall sampling of metrics summaries computation.
-    #[serde(
-        rename = "relay.compute-metrics-summaries.sample-rate",
-        deserialize_with = "default_on_error",
-        skip_serializing_if = "is_default"
-    )]
-    pub compute_metrics_summaries_sample_rate: Option<f32>,
```

Comment:
Do we need to keep forwarding this option to external relays so they keep computing the summary? Or is this option so new that we don't care?

Reply:
Discussed offline, this was only enabled for our own org so it can be removed.